src/main.py

```diff
@@ -1,7 +1,25 @@
 """The main pipeline"""
 
-from src.utils.testfunctions import add
+import toml
+from utils.testfunctions import add
 
+def toml_parser():
```

Comment:
Thinking about this function name, is it descriptive enough? Will this parse all toml files, or just one in particular? Could the name be more exact/descriptive and give clues about what it's actually doing?

Reply:
Maybe it will be a general purpose toml parser, if so, great, let's keep it like that. In that case the docstring would need to reflect its more general function

Reply:
Would something like user_config_reader be suitable?